content/en/docs/_index.md

```diff
@@ -1,20 +1,41 @@
 
 ---
-title: "Tekton Documentation"
+title: "Welcome to Tekton"
 linkTitle: "Documentation"
 weight: 20
 menu:
   main:
     weight: 20
 ---
 
-Tekton is a cloud native continuous integration and delivery (CI/CD)
+Tekton is an open-source cloud native continuous integration and delivery (CI/CD)
 solution. It allows developers to build, test, and deploy across cloud
 providers and on-premise systems.
 
-### What's next
+The Tekton project consists of the following components:  
 
-To get started, read the contents below.
+- **[Tekton Pipelines](https://github.com/tektoncd/pipeline/blob/main/docs/README.md)** provides basic building blocks (tasks and pipelines) of a CI/CD workflow.
+
+- **[Tekton Triggers](https://github.com/tektoncd/triggers/blob/main/README.md)** allows you to instantiate pipelines based on events.
+
+- **[Tekton CLI](https://github.com/tektoncd/cli/blob/main/README.md)** provides a command-line interface called `tkn`, built on top
+  of the Kubernetes CLI, that allows you to interact with Tekton.
+
```

Comment:
This is duplicating content from the Tekton Overview. We should not be duplicating content across docs because it can quickly go out of sync. Instead, let's remove this list and add a short blurb with a link to the full overview:

"Tekton is a cloud-native solution for building CI/CD pipelines. It consists of Tekton Pipelines, which provides the building blocks, and of supporting components, such as Tekton CLI and Tekton Catalog, that make Tekton a complete ecosystem. Tekton is part of the CD Foundation, a Linux Foundation project. For more information, see the [Tekton Overview](https://github.com/tektoncd/website/blob/main/content/en/docs/Overview/_index.md)."

Reply:
@sergetron Thanks a lot for your valuable comments. It truly help me to know how to improve this part.

Reply:
@AlanGreene  @sergetron  Hi, I added remove the duplicate list and add the blurb instead,  also add links with "Overview  of Tekton", "CD Foundation" and "Linux Foundation". Would you please check  it ? Thank you